providers/gce/gce.go

```diff
@@ -94,6 +94,12 @@ var _ cloudprovider.Zones = (*Cloud)(nil)
 var _ cloudprovider.PVLabeler = (*Cloud)(nil)
 var _ cloudprovider.Clusters = (*Cloud)(nil)
 
+type StackType string
+
+const NetworkStackDualStack StackType = "IPV4_IPV6"
```

Comment:
is there a reference from where this constant is coming from

Reply:
These are in the comments of the go sdk.
https://pkg.go.dev/google.golang.org/api@v0.56.0/compute/v0.alpha
(search for "IPV4_IPV6")
Unfortunately the generated code did not have the constants directly avaliable.

Reply:
This is actually being set by the GKE cluster server. This value is going to be written to gce.conf.